examples/knockback/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "private": true,
+  "dependencies": {
+    "Backbone.localStorage": "^1.1.3",
```

Comment:
Oh Sindre. Come on! Beers on me if you find again something! I guess you are printing a picture of me to use for Darts! :stuck_out_tongue: